source/loader/ur_lib.cpp

```diff
@@ -12,25 +12,63 @@
 #include "ur_lib.hpp"
 #include "logger/ur_logger.hpp"
 #include "ur_loader.hpp"
-#include "ur_proxy_layer.hpp"
-#include "validation/ur_validation_layer.hpp"
 
-#if UR_ENABLE_TRACING
-#include "tracing/ur_tracing_layer.hpp"
-#endif
+#include <cstring>
 
 namespace ur_lib {
 ///////////////////////////////////////////////////////////////////////////////
 context_t *context;
 
 ///////////////////////////////////////////////////////////////////////////////
-context_t::context_t() {}
+context_t::context_t() {
+    for (auto l : layers) {
+        if (l->isAvailable()) {
+            for (auto &layerName : l->getNames()) {
+                availableLayers += layerName + ";";
+            }
+        }
+    }
+    availableLayers.pop_back();
```

Comment:
Is this to remove the ending `;`?

I was confused at first. Can you please add a comment?